draft-ietf-trans-gossip.md

```diff
@@ -249,25 +249,30 @@ described in {{feedback-srvaud}}.
 ### HTTPS client to server {#feedback-clisrv}
 
 When an HTTPS client connects to an HTTPS server, the client receives
-a set of SCTs as part of the TLS handshake. The client MUST discard
+a set of SCTs as part of the TLS handshake, including in the server certificate. The client MUST discard
 SCTs that are not signed by a log known to the client and SHOULD store
 the remaining SCTs together with the corresponding certificate chain
 for later use in SCT Feedback.
 
-When the client later reconnects to any HTTPS server for the same
-domain, it again receives a set of SCTs. The client MUST add new SCTs
-from known logs to its store of SCTs for the server. The client MUST
-send to the server any SCTs in the store that are associated with that
-server but which were not received from that server.
+The SCTs stored on the client MUST be keyed by the exact domain name 
+the client contacted. They MUST NOT be sent to any domain related 
+to the original (e.g. if the original domain is sub.example.com they 
+must not be sent to sub.sub.exmaple.com or to example.com.) They MUST
+NOT be sent to any Subject Alternate Names specified in the certificate.
+Disobeying these security requirements will leak user's browing habits 
+cross-domain. In the case of certificates that validate multiple domain
+names, it is expected that the same SCT may be stored multiple times.
+
+When the client later connects to an HTTPS server it again receives a 
+set of SCTs. The client MUST add new SCTs from known logs to its store 
+of SCTs for the individual domain name contacted. The client MUST
+send to the server SCTs in the store that are associated with that
+domain name but which were not received from that server. 
 
```

Comment:
Can you elaborate on the last sentence, especially the last part starting with "but which ..."?